### PR TITLE
Add flake8 to tests; move /test directory

### PR DESCRIPTION
**Pull request by griffinmyers**
#### Description

* Adds flake8 lint to test suite
* Move `/test` to `/pybutton/test` (part of linting)
* Migrate relative imports to package absolute imports
* Fix existing lint errors

### CODE REVIEW

File: pybutton/test/flake8_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class Flake8Test(TestCase):
+
+    def test_flake8(self):
+
+        # As of v3, flake8 is no longer supports python 2.6
+        if sys.version_info[0] == 2 and sys.version_info[1] < 7:
+            return
+
+        command = 'flake8 pybutton'
+        p = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
```
Comment (griffinmyers) [inline pybutton/test/flake8_test.py:20]: `Popen` used in favor of `check_output` to maximize python compatibility.  (didn't find out about 2.6 support being dropped by flake8 until after this was in place; and, no harm leaving.)

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F821
```
Comment (griffinmyers) [inline .flake8:2]: F821 complains about `unicode` being referenced in python `>=3`

**Review by danielmcgrath — APPROVED** [review]
LGTM!

**Comment by griffinmyers** [comment]
thanks, @danielmcgrath !

**Pull request merged**